CONTRIBUTING.md

```diff
@@ -145,7 +145,9 @@ General guidelines for making a good gallery plot:
   plot the data*.
 * Describe the feature that you're showcasing and link to other relevant parts of the
   documentation.
-* SI units should be used in the example code for gallery plots.
+* SI units should be used in the example code for gallery plots and should follow the
```

Comment:
I agree these are good rules, and the changes in this PR are all good, but I'm still inclined to remove these rules to keep it simple.

Reply:
To clarify, do you mean removing the entire bullet point or just the added section about the NIST standards? I am asking because the SI units rule was in the contributing guide prior to this PR.

I prefer explicit guidelines, but I am fine with deferring to yours and other's preferences here.

Reply:
I meant keeping "SI units should be used in the example code for gallery plots" only, since these rules are applied to all example scripts and the tests.

The NIST rules are only applied to comments and some labels in figures. That's what I want to remove.

Reply:
I'm fine with removing the sections including the NIST content to keep the whole thing simple. Thanks for the detailed discussion and your comments @seisman & @meghanrjones.